velox/expression/fuzzer/ExpressionFuzzer.cpp

```diff
@@ -341,32 +341,6 @@ static void appendSpecialForms(
   }
 }
 
-/// Returns if `functionName` with the given `argTypes` is deterministic.
-/// Returns true if the function was not found or determinism cannot be
-/// established.
-bool isDeterministic(
```

Comment:
Let's keep this function, but remove 'argTypes' and replace code from L359 with a call to new isDeterministic(name) API. The new API would go into elox/functions/FunctionRegistry.h and return `std::optional<bool>`. std::nullopt would indicate that function not found, true would indicate that all function signatures are deterministic, false would mean that at least one function signature is non-deterministic.

What do you think?

Reply:
Thanks for the suggestion. Added a new isDeterministic API in FunctionRegistry.h.